core/src/components/menu/menu.tsx

```diff
@@ -288,23 +312,14 @@ export class Menu implements ComponentInterface, MenuI {
   }
 
   @Listen('ionSplitPaneVisible', { target: 'body' })
-  onSplitPaneChanged(ev: CustomEvent) {
-    const { target } = ev;
-    const closestSplitPane = this.el.closest('ion-split-pane');
+  async onSplitPaneChanged(ev: CustomEvent<{ visible: boolean }>) {
```

Comment:
Why was this made async? It doesn't `await` anything.

Reply:
Good catch! That was left over from some older attempts at fixing the issue. Removed in https://github.com/ionic-team/ionic-framework/pull/28801/commits/166df6c3922fba3f5d3d6c28737c8cfebbf6285d